jax/interpreters/sharded_jit.py

```diff
@@ -73,13 +74,28 @@ def _sharded_callable(
     fun: lu.WrappedFun, num_partitions: Optional[int],
     in_parts: Tuple[pxla.PartitionsOrReplicated, ...],
     out_parts_thunk: Callable[[], Tuple[pxla.PartitionsOrReplicated, ...]],
-    name: str, *abstract_args):
+    local_in_parts: Optional[Tuple[pxla.PartitionsOrReplicated, ...]],
+    local_out_parts_thunk: Callable[[], Optional[Tuple[pxla.PartitionsOrReplicated, ...]]],
+    local_num_parts: Optional[int], name: str, *abstract_args):
```

Comment:
It doesn't really matter, but I was tripped up once or twice by the fact that these parameters are called "num_parts" in some places and "num_partitions" in others.

Reply:
You're right, this is super inconsistent. I adopted the pxla convention and changed everything to `nparts` and `local_nparts` (for brevity!), except the argument names to `sharded_jit`, which I kept as [local_]num_partitions.